unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/FavoritesSubSection/IFavoritesSubSectionComponentController.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+public interface IFavoritesSubSectionComponentController : IDisposable
```

Comment:
Don't see clients of this interface. Remove it or create test-cases with mocks, that will use it

Reply:
I'm not yet making the test cases because the data structure still needs to be defined and the functions in there are to define the overall structure, in the next few days as soon as we have defined some additional things I'll add them :)